db/migrate/20190325064225_create_accepted_comment_likes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateAcceptedCommentLikes < ActiveRecord::Migration[5.2]
+  def change
+    def up
+      answers = Answer.where(accepted: true)
+      timestamp_to_answer_id = answers.group('created_at').minimum('id')
+                                         .inject({}) {|hash, (key, value)| hash[key.to_i] = value; hash}
+
+      Comment.where(timestamp: timestamp_to_answer_id.keys).or(Comment.where(comment: answers.pluck(:content))).each do |c|
+        c.likes.create(emoji_type: "Accepted")
```

Comment:
Ah! Capitalization? Above this is `"accepted"`, no?

Reply:
By the convention we are following it has to be Accepted, while comparison and all, it is converted to downcase so it isn't a problem.